src/components/common/FormControl/FormControlStyles.tsx

```diff
@@ -0,0 +1,359 @@
+import SVG from 'react-inlinesvg'
+import styled, { DefaultTheme } from 'styled-components'
+
+import {
+	ComponentSize,
+	ComponentSizeSmallMediumLarge
+} from '../../../enum/ComponentSize'
+import { ColorTheme } from '../../../theme/ColorTheme'
+
+/** Customizable */
+export const INPUT_PADDING_X = 11
+
+/**
+ * Common constants and functions
+ */
+
+const LEGEND_BORDER_PADDING_X = INPUT_PADDING_X - 7
```

Comment:
Well, it should be actually in the theme configuration file. We already have one. I'll move it there.

Reply:
Is the theme editable from outside the package itself? Meaning - can developers using phoenix components modify the theme from within their project?

Reply:
Developers (our developers) should not edit the theme file. The theme file represents variables (colours, sizes etc.) that reflect the Design System. We're trying to put it in a standalone file so that someone can fork our project and modify the theme for their own needs (their own brand colours etc.)

Reply:
We've realized with @samuelg0rd0n  that users of phoenix-components need to supply styled components theme anyway.. so we can move all this configuration there.

Reply:
Constants moved to theme and the whole theme prefixed with `$pc`.